### ISSUE TITLE
Improve 404 not found page

### ISSUE THREAD

**Issue by scorpiontornado** [coopsoc/website#23]
Currently has a black background and dark purple text, clashing with the rest of the website

**Comment by scorpiontornado** [comment]
To replicate, go to [https://coopsoc.com.au/a](https://coopsoc.com.au/a)

Should just be a simple change to the CSS (change background colour to white, and possibly text colour, text size, and/or position of the 404 message). Could add a little question mark icon if you're feeling fancy. Also change the colour of "Copyright © 2022 UNSW Co-op Society" to black to be consistent with the other pages.

**Comment by vishalpawarR** [comment]
@scorpiontornado I want help can you let me know where to make change in which file?? Beginner OS contributor!

**Comment by theawesomew** [comment]
If you're interested in making a contribution, you're very welcome to fork the repository and make a pull-request. I'm the administrator of this repository and I welcome any and all contributions!

[This link should help get you started :)](https://www.slingacademy.com/article/how-to-create-a-custom-404-not-found-page-in-next-js/#:~:text=The%20404%20error%20means%20the,links%20to%20incorrect%20server%20configuration.)

**Comment by lhvy** [comment]
Nick and I started working on this. Currently have a new 404 page created, just making it look ✨pretty✨.

**Comment by scorpiontornado** [comment]
> @scorpiontornado I want help can you let me know where to make change in which file?? Beginner OS contributor!

Sorry, this turned out to be a bit more complicated than I was expected (turns out its currently using the default next.js 404 page, so its not just changing a small bit of CSS like I thought). We're working on it now, but feel free to have a go at another issue!

**Comment by lhvy** [comment]
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/40507205/231661806-ced57d1d-6488-46a0-a595-aa64ba59246d.png">

Thoughts on this 404 page? I added the return home button, but I'm wondering if I could just make the 404 text/icon larger and simply let the user click on any of the items in the nav bar instead.

**Comment by scorpiontornado** [comment]
Yeah I don't think we need an explicit return home button, the logo in the nav bar already does that (might be a bit unintuitive for some users but I think most ppl know to click on a logo or one of the other pages in the nav bar). Some small nitpicks though - maybe add a bit more padding between the not found message and the magnifying glass. Also, you could either centre both the message and magnifying glass vertically, or add a bit more padding before the message. Could also consider making the magnifying glass bigger and/or putting it before the message.

**Comment by theawesomew** [comment]
I agree with Nick's idea to add more padding below the 404 PAGE NOT FOUND text but, I disagree with removing the 'return to home' button - I feel that's a more ergonomic way of addressing people thought of 'Oh, where do I go next then?' which is - to me - a natural question after finding that the page you're looking for no longer exists or never did. So, leave that there - otherwise, it's looking pretty good currently.

**Comment by scorpiontornado** [comment]
Yo @lhvy could you MR this

**Comment by lhvy** [comment]
Yeah sure, not sure what the holdup was, I'll have to check again but I should have something good enough anyway. (Also lol someone's spent too long on GitLab)

**Comment by scorpiontornado** [comment]
@lhvy could you check if this is ready to merge?

**Comment by scorpiontornado** [comment]
Think we've lost this code, will have to rewrite from scratch. Shouldn't be too hard (hopefully)
